test/local/push_db_tests.js

```diff
@@ -101,6 +102,7 @@ test(
         })
         .then(function(emailRecord) {
           emailRecord.createdAt = Date.now()
+          emailRecord.tokenVerificationId = ACCOUNT.tokenVerificationId
```

Comment:
It's not obvious why we need to add zeroed-out tokenVerificationIds here - can't we create the token with no id and have things continue to work?